dragonfly/room2d.py

```diff
@@ -756,7 +756,7 @@ def remove_colinear_vertices(self, tolerance=0.01):
         rebuilt_room._properties._duplicate_extension_attr(self._properties)
         return rebuilt_room
 
-    def to_honeybee(self, multiplier=1, tolerance=0.01):
+    def to_honeybee(self, multiplier=1, tolerance=0.01, add_plenums=False, plenum_ht=1.0):
```

Comment:
Also, this is a minor styling comments but, for input arguments, I usually like to write out the full word (eg. `plenum_height`) instead of using abbreviations. Inside the method, I use abbreviations a lot but the inputs are going to be read by more than just the source code developers so I feel it's better to be explicit.

Reply:
@chriswmackey what's the best way to distinguish or combine floor and ceiling plenums. Should I add a `add_floor_plenum` AND a `add_ceiling_plenum` argument to the method, or should I just assume there's always a floor and ceiling plenum. 

I think it makes sense to have just a ceiling plenum set as default, and having the floor plenum as an option (from a modeler's perspective) since that is the most common plenum scenario. So something like: 
```
to_honeybee(self, .... add_ceiling_plenum=True, add_floor_plenum=False)
```

?

Reply:
@chriswmackey 

As I'm implementing the floor_plenum, I realize I don't know what is the best rule to autocalculate floor plenum height. Should it be 1/2 the distance we calculate from parent.floor_to_floor - self.floor_to_ceiling? This would mean that when floor plenums are set, we are assuming there's a ceiling plenum that's filling the rest of the gap between vertically adjacent rooms. 

Alternatively it can be the full height of the story/room gap (like how we have the ceiling plenum height right now) - but that means that we can only have a floor OR ceiling plenum, but never both. 

What's the desired behaviour?